### PR TITLE
docs(layout): add usage example of `templateAreas`

### PR DESCRIPTION
**Pull request by aroyan**
## 📝 Description

> Add usage example of `templateAreas` prop on `<Grid /> component.

## ⛳️ Current behavior (updates)

> There is no example of how to use `templateArea` prop

## 🚀 New behavior

> Add example of how to use `templateArea` prop

## 💣 Is this a breaking change (Yes/No):

> No

### CODE REVIEW

**Review by nikolovlazar — CHANGES_REQUESTED** [review]
Thanks for opening this PR, @aroyan! It's looking good! I would just change the backgrounds to a "fresher" shade, as opposed to the `900` shade 😅

**Comment by aroyan** [comment]
I realized it's a bad color choices, I just want the text to be seen. Or should I just delete those text inside `<GridItem/>`?

File: pages/docs/components/layout/grid.mdx [hunk 1]
```diff
@@ -75,6 +75,29 @@ start or end at the `nth` grid position.
...
+                  "nav footer"`}
+  gridTemplateRows={'50px 1fr 30px'}
+  gridTemplateColumns={'150px 1fr'}
+  h='200px'
+  gap={1}
+>
+  <GridItem bg='orange.300' area={'header'} />
+  <GridItem bg='pink.300' area={'nav'} />
+  <GridItem bg='green.300' area={'main'} />
+  <GridItem bg='blue.300' area={'footer'} />
```
Comment (nikolovlazar) [inline pages/docs/components/layout/grid.mdx:97]: I think it's good to leave the text. To make it more visible, you can play with the font size and weight.

**Review by nikolovlazar — CHANGES_REQUESTED** [review]

File: pages/docs/components/layout/grid.mdx (hunk 1 above)
Comment (aroyan) [reply pages/docs/components/layout/grid.mdx:97]: Working on it 💻

**Review by nikolovlazar — APPROVED** [review]
Love it! Thanks, @aroyan!

**Pull request merged**